server/cron/cron.go

```diff
@@ -121,7 +122,23 @@ func CreatePipeline(ctx context.Context, store store.Store, forge forge.Forge, c
 		return nil, nil, err
 	}
 
-	commit, err := forge.BranchHead(ctx, creator, repo, cron.Branch)
+	// if the forge has a refresh token, the current access token
+	// may be stale. Therefore, we should refresh prior to dispatching
+	// the pipeline.
+	if refresher, ok := server.Config.Services.Forge.(forge.Refresher); ok {
```

Comment:
```suggestion
	if refresher, ok := f.(forge.Refresher); ok {
```
shouldnt this also work?

Reply:
It does :-)

Updated